src/index.tsx

```diff
@@ -15,10 +15,10 @@ export {
   SkinTones,
   Theme,
   Categories,
-  EmojiClickData,
   SuggestionMode,
   SkinTonePickerLocation
 } from './types/exposedTypes';
+export type { EmojiClickData } from './types/exposedTypes';
```

Comment:
@ealush this won't work without adding the type anotation and moving it to diffrent export not sure if it actually affecting the package types but I think it not